### DISCUSSION TITLE
Regarding career in IT

### DISCUSSION THREAD

**Discussion by akash631** [praveenscience/praveenscience#13] [Careers]
Hello sir this is Akash final year cse engineering student. Can you please suggest me any field in IT without coding  which are in demand and will be interesting. I need any career advice for starting a new job in IT.

**Comment by praveenscience** [comment]
Hi Akash, thanks for the question. If coding isn't your cup of tea, that's completely fine. There are so many different options in IT field without coding! Starting with this No Code movement, you can develop websites and other things without coding at all. Other areas you can be a part of and earn a lot are:

* Graphic Designing
* Cloud Computing
* Deploying Website
* Setting up WordPress / Shopify, etc.

There are so many you can try out! 😁 Let me know if those might help you. I can even get you some tutorials on the above. Everything might get you a huge money + you don't need to write a single code! 😁

**Reply by akash631** [reply]
Thanks Can you help me out i.e from where should I start from scratch of the above mentioned fields where to learn those and how to apply for the jobs you mentioned which is easy one to learn

**Reply by praveenscience** [reply]
Everything is available for you in YouTube - beginners to advanced. I can suggest you a couple of them right now:

* Graphic Designing
   * [Graphic Design Tutorials](https://www.youtube.com/playlist?list=PL9pkETrdJ0rZoid7XU1OmQ5zvPVQ0sRJl)
   * [Graphic Design Theory & Techniques](https://www.youtube.com/playlist?list=PL-c9Rq56P4Km_9gvxoNfVK3Jvh9ynGwgl)
   * [Beginners guide to Graphic Design | 45 Episode FREE Series](https://www.youtube.com/playlist?list=PLYfCBK8IplO4E2sXtdKMVpKJZRBEoMvpn)
* Cloud Computing
   * [Cloud Computing Tutorial For Beginners](https://www.youtube.com/playlist?list=PLEiEAq2VkUUIJ3o1tehvtux0_Ynf42CBN)
   * [Microsoft Azure Training Videos](https://www.youtube.com/playlist?list=PL9ooVrP1hQOHdFketT6JzY-71nBgIu-n0)
* Deploying Website & Web Development
   * [30 Days of React](https://rb.gy/cxk3ek)
   * [Winter of Code](https://www.youtube.com/playlist?list=PLO_Y0rsm7b3Yc8FtNhCuD5TO0QM8CnTZj)
   * [Portfolio Building Beginners](https://www.youtube.com/playlist?list=PLO_Y0rsm7b3Y-TzqfEJZYWi3HCO_eCgzW)
   * [Become Full Stack Developer - Zero to Hero](https://www.youtube.com/playlist?list=PLO_Y0rsm7b3Yjj-BCNcrDlH3gx_tHZDLQ)
* Setting up WordPress / Shopify, etc.
   * [How to install WooCommerce | Fresh install](https://www.youtube.com/watch?v=hRDuQtmzq3M)
   * [Step-by-Step WordPress Setup on Linux, Nginx, MySQL, and PHP/php-fpm](https://www.youtube.com/watch?v=4beEybPzYqQ)
   * [How To Make a WordPress Website](https://www.youtube.com/watch?v=VnjeR-bsRM4)

Hope these help you start with...

**Reply by akash631** [reply]
Thank you so much for your valuable guidance
